### PR TITLE
Automatically create a changelog in open-brush-docs

### PR DESCRIPTION
**Pull request by mikeage**

### CODE REVIEW

**Comment by mikeage** [comment]
Replaced #414, since I needed it to be from this repo in order to test the secret

**Comment by mikeage** [comment]
Sample available at https://github.com/icosa-gallery/open-brush-docs/blob/master/alternate-and-experimental-builds/automatic-release-notes/automatic-changelog.md

The "to" tag will be correctly populated (I hope!) once this is merged; because I'm now testing it from a PR, the target is a bit wonky looking.

Please review this, but DO NOT MERGE IT; I need to remove a commit first, but I want to leave it there for now in case any changes are requested.

**Review by andybak — APPROVED** [review]

**Pull request merged**
